helmfile/charts/networkpolicy-service/templates/fluentd/forwarder.yaml

```
@@ -0,0 +1,43 @@
+{{ if .Values.fluentd.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-fluentd-forwarder
+  namespace: fluentd
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: forwarder
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              app.kubernetes.io/name: prometheus
+              app.kubernetes.io/instance: kube-prometheus-stack-prometheus
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+      ports:
+        - port: 24231
```

```suggestion
        - port: 24231
    - from:
      {{- if .Values.global.ingressUsingHostNetwork }}
        {{- range $ip := .Values.global.scNodes.ips }}
        - ipBlock:
            cidr: {{ $ip }}
        {{- end }}
      {{- else }}
        - namespaceSelector:
            matchLabels:
              kubernetes.io/metadata.name: ingress-nginx
          podSelector:
            matchLabels:
              app.kubernetes.io/component: controller
              app.kubernetes.io/instance: ingress-nginx
      {{- end }}
      ports:
        - port: 9980
```
I needed to add this to my setup
Or atleast saw that one of the nodes was talking to the fluentd pods on this port